bot.js

```diff
@@ -219,6 +221,34 @@ client.on("guildMemberUpdate", memberUpdateHandler);
 // Every hour check if the current time is within the ranges
 setInterval(() => {
   const shouldPost = isRedditPostingTime();
+  if (!shouldPost) return;
+    Phin({ url: REDDIT_URL, parse: "json" })
+      .then((res) => {
+          notSticky(res);
+          
+      })
+   
 }, 60 * 60 * 1000);
 
+// checks to make sure that the top post is not a sticky post
+function notSticky(res){
+  try {
+      const posts = res.body.data.children;
+
+      for (let i = 0; i < posts.length; i++) {
+          if (posts[i].data.stickied === true) continue;
+          else {
+              console.log(`https://reddit.com${posts[i].data.permalink}`)
```

Comment:
Let's transition this function into returning the first not stickied post. Change function name and usage appropriately.